patch/kernel/archive/bcm2711-5.19/1003-remote-wakeup.patch

```diff
@@ -0,0 +1,160 @@
+diff --git a/drivers/usb/dwc2/gadget.c b/drivers/usb/dwc2/gadget.c
```

Comment:
@The-going This works on Orangepi Zero ... This is needed for PiKVM to run there. Any concerns?

Reply:
It's pretty hard to tell looking at the patch.
I usually do a "get rid of or make not working" code that controls the device falling asleep \ waking up. That is, the task is reversed.